xml/ha_resource_constraints.xml

```diff
@@ -1241,6 +1241,16 @@
       params autoset_utilization_host_memory=false \
       op monitor timeout=30s interval=10s</command></screen>
     </example>
+    <para>
+     For either virtualization type, make sure to clone the <systemitem>NodeUtilization</systemitem>
+     resource so it can run on all nodes:
+    </para>
+<screen>&prompt.root;<command>crm configure cl_nu p_nu</command></screen>
+    <para>
+     We also recommend setting the <literal>placement-strategy</literal> property to
+     <literal>balanced</literal>:
```

Comment:
I'd like to rephrase as 

"""
The following command set the `placement-strategy` property to `balanced`.
"""

I personally have fewer recommendations for its value. The end user has their own free choice.

Reply:
Thanks! That works for me :+1: